### ISSUE TITLE
Add information about monitoring

### ISSUE THREAD

**Issue by kcpevey** [Quansight/qhub#255]
I've had several env build failures recently (various reasons) that weren't caught by ci testing. On multiple occasions we've had to go into `kubectl`/`k9s` to see the failure or were it got hung up. 

I like how everything is automated here, but when my gh action for deployment is showing "green" its misleading that the deployment is complete. I'd like some tooling (or documentation) to tell me how to set up `kubectl` to see what's going on after gh actions has done it's thing. 

I know the pretense is that qhub is k8s for people who don't know k8s, but I need some additional monitoring capability.

**Comment by tylerpotts** [comment]
> I know the pretense is that qhub is k8s for people who don't know k8s, but I need some additional monitoring capability.

@kcpevey I don't think this captures the purpose of qhub. Ideally, people shouldn't need to be aware qhub is backed by kubernetes. Ideally, they add a few API keys, and :boom: , a stable data science platform is created

This issue is also related to #86 

Better logging of conda environments that are being created is a feature we have yet to implement. Changing the title to reflect this: `Add logging for in progress conda environments builds`

While it's possible to see logs a[s explained here](https://github.com/Quansight/qhub/issues/242#issuecomment-761035588), it's certainly not user friendly. I also don't think it's real-time (IE the logs get dumped once the conda solve either finished or failed)
